src/components/App/AppPages.component.js

```diff
@@ -16,6 +17,7 @@ export const AppPages = () => (
         <Route path="/new" component={NewPage} />
         <Route path="/enrollment/stageEvents" component={StageEventListPage} />
         <Route path="/enrollment/event/edit" component={EnrollmentEventPage} />
+        <Route path="/enrollment/event/new" component={EnrollmentAddEventPage} />
```

Comment:
Hey, 
I suggest you use a similar pattern `/enrollmentEventNew` for the URL like this one [here](https://github.com/dhis2/capture-app/pull/1803/files#diff-1234f6997697c9268f5cec67e82675b26b9ff38bb6662ac18a6e42ad0440a174). This will be used by [withAppUrlSync](https://github.com/dhis2/capture-app/pull/1803/files#diff-d056d752047e849da36a2582e9101804325a15da7ce1afd642076c7322f8db8e) in the next tickets when the event form is added to the page.

Reply:
hey @simonadomnisoru, thank you for letting me know.
Is this pattern decided and validated? Because in my ticket the url is `/enrollment/event/new` and QA has need to be notified this change in order to perform the test

Reply:
Last time I spoke with @JoakimSM about the URL pattern he agreed with the change. The PR where I did this change is waiting for him to validate other parts so it can be merged.
To notice QA, you just need to update the Jira ticket DHIS2-11388 description and its test cases ticket with the new URL.